src/Cedar/Cedar.h

```diff
@@ -1024,6 +1027,7 @@ typedef struct CEDAR
 	COUNTER *ConnectionIncrement;	// Connection increment counter
 	X *ServerX;						// Server certificate
 	K *ServerK;						// Private key of the server certificate
+	char UsernameHubSeparator;		// Character which separates the username from the hub name

```

Comment:
does it allow unicode characters ?

Reply:
No, it doesn't.

Reply:
what will happen if user will input unicode char?

Reply:
`@` is used, we check whether the character is usable with `IsPrintableAsciiChar()`.